internal/cmd/asyncapi/command_export.go

```diff
@@ -279,35 +272,16 @@ func (c command) getMessageExamples(consumer *ckgo.Consumer, topicName, contentT
 	return jsonMessage, nil
 }
 
-func (c *command) getBindings(cluster *ccstructs.KafkaCluster, topicDescription kafkarestv3.TopicData) (*bindings, error) {
-	kafkaREST, err := c.GetKafkaREST()
-	if err != nil {
-		return nil, err
-	}
```

Comment:
Isn't this how most commands use Kafka REST? Why are you choosing to do it a different way?

Reply:
From this [slack thread](https://confluent.slack.com/archives/CM6PEBEGM/p1678366399353049?thread_ts=1678357423.658659&cid=CM6PEBEGM): it seems like GetKafkaREST() tries to set up a new connection every time. I wanted to establish it once and use the same one for each topic.

Reply:
On going through it again, it seems the error is coming from ListKafkaTopicConfigs(), so switching it back.